Linux/Car_Client/chronos.cpp

```diff
@@ -61,13 +62,25 @@ void Chronos::startTimerSlot()
 {
     qDebug() << "Starting car";
     mObjectState = ISO_OBJECT_STATE_RUNNING;
-
+	mChronos->startHeabLastHeabReceivedTimer();
```

Comment:
I just noticed that there is some kind of delayed start option in Chronos::processStrt(). Have you thought about starting the timer there instead? I don't know anything about this feature, it's just a thought.

Reply:
Good catch. We have the option of sending START messages with a delay in ms, it might be better to move this.